src/tokens.yaml

```diff
@@ -32,8 +34,12 @@ radius:
     medium: 10px
     small: 5px
 shadow:
-    dark: "rgba(0, 0, 0, 0)"
-    light: "#f0f3ff"
+    tile:
+        dark: "rgba(255,255,255,.14)"
+        light: "rgba(0,0,0,.14)"
```

Comment:
A lot of duplicates here, can they be pulled out into their own token

Reply:
Removed unneeded shadow tokens